x.ps1

```diff
@@ -24,5 +28,12 @@ foreach ($python in "py", "python3", "python", "python2") {
     }
 }
 
+$found = (Get-Application "python*" | Where-Object {$_.name -match '^python\d'})
```

Comment:
This will also pull in things like `python3-config`. Let's limit it to the exact M.m format you called out in the PR description.
```suggestion
$found = (Get-Application "python*" | Where-Object {$_.name -match '^python[2-3]\.[0-9]$'})
```

Reply:
This exact match excludes `python3.10`, and on Windows this also excludes the executables, as they all end with a `.exe` extension.

Reply:
Ok, try this regex instead:
```suggestion
$found = (Get-Application "python*" | Where-Object {$_.name -match '^python[2-3]\.[0-9][0-9]?(\.exe)?$'})
```
We can't merge this as is, it will break people with `python-config` installed.